cmd/xl-v1-healing.go

```diff
@@ -599,6 +603,18 @@ func (xl xlObjects) isObjectDangling(metaArr []xlMetaV1, errs []error) (validMet
 			corruptedXLJSON++
 		}
 	}
+	var notFoundParts int
+	for i, readErr := range dataErrs {
+		// Only count part errors, if the error is not
+		// same as xl.json error. This is to avoid
+		// double counting when both parts and xl.json
+		// are not available.
+		if errs[i] != dataErrs[i] {
```

Comment:
`Is `if errs[i] != dataErrs[i] {` same as `if errs[i] != readErr {`

Reply:
Its to convey a message @kannappanr to be more readable, I would perhaps deprecate `readErr`

Reply:
Now, it looks cleaner to me.